README.md

```diff
@@ -272,6 +272,93 @@ the credentials provided during the seed.
 
 All set! Start playing around with the API using the Swagger UI below to learn more about the available endpoints. Enjoy!
 
+### Clustering
```

Comment:
Since this is an advanced topic, I believe it should be in a specific file in `/docs/` and referenced in the readme.